build/packaging/linux/bin/kbnd

```diff
@@ -20,6 +21,17 @@ __pid_run() {
     fi
 }
 
+__auto_restart_daemon_pid_run() {
+    unset auto_restart_daemon_pid
+    if [ ! -f $auto_restart_daemon_pidfile ]; then
+        return
+    fi
+    AUTO_RESTART_DAEMON_PID_NUM=$(eval "cat $auto_restart_daemon_pidfile")
+    if [[ ! -z "$AUTO_RESTART_DAEMON_PID_NUM" ]]; then
+        export auto_restart_daemon_pid=$(eval "ps -p $AUTO_RESTART_DAEMON_PID_NUM -o pid=")
```

Comment:
Is it necessary to use `$(eval "cmd...")` instead of `$(cmd...)`?

It seems like this pattern was already in use in other places, but I think `$(cmd)` should have the same result (capturing output of the command to an environment variable) while being simpler to read and avoiding the use of `eval`.

Reply:
@aluminous 
I removed eval. thanks.